libdnf/base/goal.cpp

```diff
@@ -37,11 +41,88 @@ void add_obseletes(const libdnf::rpm::SolvQuery & base_query, libdnf::rpm::Packa
     data |= obsoletes_query;
 }
 
+
 }  // namespace
 
 
 namespace libdnf {
 
+namespace {
+
+static const std::map<ProblemRules, const char *> PKG_PROBLEMS_DICT = {
+    {ProblemRules::RULE_DISTUPGRADE, _("{} does not belong to a distupgrade repository")},
```

Comment:
This is not a good approach to translation. The translation function _ () will be called too soon - during the initialization of the PKG_PROBLEMS_DICT map. Usually the locale is not set yet. So the translation will not work.
Translation is not the point of this PR. So you can leave it as it is. And I can prepare new PR with translation support.

Please add TODO. Translation must be done later. After setting the locale.